.gitignore

```diff
@@ -170,3 +170,6 @@ docs/ExampleNotebooks/
 
 # duecredit
 .duecredit.p
+
+# Some charge stuff
+*.model.pt
```

Comment:
Either nagl or espaloma is creating these

Reply:
Looks like it's espaloma: https://github.com/choderalab/espaloma_charge/issues/30